packages/api/cms-api/src/warnings/warning.resolver.ts

```diff
@@ -99,4 +102,17 @@ export class WarningResolver {
         const [entities, totalCount] = await this.repository.findAndCount(where, options);
         return new PaginatedWarnings(entities, totalCount);
     }
+
+    @ResolveField(() => EntityInfoObject, { nullable: true })
+    async entityInfo(@Parent() warning: Warning): Promise<EntityInfoObject | undefined> {
+        const repository = this.entityManager.getRepository(warning.sourceInfo.rootEntityName);
+        const instance = await repository.findOne({ [warning.sourceInfo.rootPrimaryKey]: warning.sourceInfo.targetId });
+
+        if (instance) {
+            const entityInfo = await this.entityInfoService.getEntityInfo(instance);
+            return entityInfo;
+        }
+
+        return undefined;
```

Comment:
This is done for each entry in the request list, but I guess it's fast enough for a live query and probably not worth optimizing, is it?